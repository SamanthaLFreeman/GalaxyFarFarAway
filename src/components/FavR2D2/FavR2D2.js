import React from 'react';
import { NavLink } from 'react-router-dom';
import './FavR2D2.scss';

const FavR2D2 = ({numOfFavs}) => {
  return (
    <section className='fav-R2D2'>
      <NavLink to='/favorites'><button className='favorites-btn'>View Favorites {numOfFavs}</button></NavLink>
    </section>

  );
}

export default FavR2D2;