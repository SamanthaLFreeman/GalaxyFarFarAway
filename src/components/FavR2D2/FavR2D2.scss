@import url("https://fonts.googleapis.com/css?family=Jura:500,700&display=swap");

.fav-R2D2 {
  display: flex;
  justify-content: center;
  margin-top: 20px;
  width: 100%;
}

.favorites-btn {
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: #2d445b;
  border-radius: 11px 11px;
  border: hidden;
  font-size: 1em;
  height: 36px;
  width: 100%;
  color: aliceblue;
  box-shadow: 0 0 8px #fff, -1px 0 8px rgb(255, 140, 0),
    1px 0 8px rgb(0, 30, 255);
}

.star-img {
  height: 35px;
}

.favorites-btn:hover {
  box-shadow: 0 0 15px #fff, -3px 0 18px hsl(134, 100%, 50%),
    3px 0 18px hsl(134, 100%, 50%);
}
