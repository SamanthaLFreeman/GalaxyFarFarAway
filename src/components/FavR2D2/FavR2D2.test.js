import React from 'react';
import { shallow } from 'enzyme';
import FavR2D2 from './FavR2D2';

describe('FavR2D2', () => {
  it('should match the snapshot with all the data passed in correctly', () => {
    const wrapper = shallow(<FavR2D2 />);
    expect(wrapper).toMatchSnapshot();
  });
});