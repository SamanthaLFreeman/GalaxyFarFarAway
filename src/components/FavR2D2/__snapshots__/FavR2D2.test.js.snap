// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavR2D2 should match the snapshot with all the data passed in correctly 1`] = `
<section
  className="fav-R2D2"
>
  <NavLink
    to="/favorites"
  >
    <button
      className="favorites-btn"
    >
      View Favorites 
    </button>
  </NavLink>
</section>
`;
